portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,7 +54,12 @@
   private static final String NAME = "name";
   private static final String COMMENT_TEXT = "commentText";
   private static final String EMAIL = "email";
-  
+
+  /** Supported image files */
```

Comment:
I wouldn't make comments for private members JavaDocs. See: https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
done